### PR TITLE
Pin holoviz packages to param<2

### PR DESCRIPTION
**Pull request by maximlt**
[Param 2](https://github.com/holoviz/param) is going to be released in the next few days. Param is a foundational library of the [HoloViz ecosystem](https://holoviz.org) and is going through a major release, it comes with breaking changes that can result in a very bad experience if Param 2 is installed with versions not supporting it, as Param's machinery is exerted on class creation and if it fails leads to errors on import.

This PR aims to patch all the HoloViz packages depending on Param to pin them to `param <2`. Ideally, and if this PR is approved of course, it would be merged before Param 2 is released to make sure that users don't create environments with older and incompatible HoloViz packages and Param 2. The next releases of these packages will pin `param >=2,<3`.

A similar undertaking has been already merged on conda-forge https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/565.

<details>
<summary>`python test-hotfix.py main --subdir linux-64 noarch --show-pkgs --use-cache`</summary>

```
Creating channel directory structure for channel 'main' and platforms ['linux-64', 'noarch']
Applying url to alias main. channel_base_url='https://repo.anaconda.com/pkgs/main'
Using cache for linux-64 noarch.
Executing hotfix for channel 'main'.
Completed hotfix for channel 'main'.
Analyzing results...
Writing out new repodata as main/linux-64/repodata-patched.json for 'linux-64' platform.
New Hot Fixes:
linux-64::datashader-0.14.1-py310h06a4308_0.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
linux-64::datashader-0.14.1-py310h06a4308_1.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
linux-64::datashader-0.14.1-py37h06a4308_0.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
linux-64::datashader-0.14.1-py37h06a4308_1.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
linux-64::datashader-0.14.1-py38h06a4308_0.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
linux-64::datashader-0.14.1-py38h06a4308_1.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
linux-64::datashader-0.14.1-py39h06a4308_0.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
linux-64::datashader-0.14.1-py39h06a4308_1.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
linux-64::datashader-0.14.2-py310h06a4308_0.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
linux-64::datashader-0.14.2-py37h06a4308_0.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
linux-64::datashader-0.14.2-py38h06a4308_0.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
linux-64::datashader-0.14.2-py39h06a4308_0.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
linux-64::datashader-0.14.3-py310h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.14.3-py37h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.14.3-py38h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.14.3-py39h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.14.4-py310h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.14.4-py310h06a4308_1.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.14.4-py311h06a4308_1.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.14.4-py37h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.14.4-py38h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.14.4-py38h06a4308_1.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.14.4-py39h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.14.4-py39h06a4308_1.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.15.0-py310h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.15.0-py311h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.15.0-py38h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.15.0-py39h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.15.1-py310h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.15.1-py311h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.15.1-py38h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.15.1-py39h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.15.2-py310h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.15.2-py311h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.15.2-py38h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.15.2-py39h06a4308_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::datashader-0.6.6-py27_0.tar.bz2
-    "param >=1.6.0",
+    "param >=1.6.0,<2.0.0a0",
linux-64::datashader-0.6.6-py27_1.tar.bz2
-    "param >=1.6.0",
+    "param >=1.6.0,<2.0.0a0",
linux-64::datashader-0.6.6-py35_0.tar.bz2
-    "param >=1.6.0",
+    "param >=1.6.0,<2.0.0a0",
linux-64::datashader-0.6.6-py35_1.tar.bz2
-    "param >=1.6.0",
+    "param >=1.6.0,<2.0.0a0",
linux-64::datashader-0.6.6-py36_0.tar.bz2
-    "param >=1.6.0",
+    "param >=1.6.0,<2.0.0a0",
linux-64::datashader-0.6.6-py36_1.tar.bz2
-    "param >=1.6.0",
+    "param >=1.6.0,<2.0.0a0",
linux-64::holoviews-1.10.0-py27_1.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::holoviews-1.10.0-py35_1.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::holoviews-1.10.0-py36_1.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::holoviews-1.16.0-py310h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.16.0-py311h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.16.0-py38h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.16.0-py39h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.16.1-py310h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.16.1-py311h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.16.1-py38h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.16.1-py39h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.16.2-py310h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.16.2-py311h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.16.2-py38h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.16.2-py39h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.17.0-py310h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.17.0-py311h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.17.0-py38h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.17.0-py39h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.17.1-py310h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.17.1-py311h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.17.1-py38h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.17.1-py39h06a4308_0.tar.bz2
-    "param >=1.12.0,<3.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::holoviews-1.9.1-py27h8db4e1f_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::holoviews-1.9.1-py35hd7d2f6e_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::holoviews-1.9.1-py36h76d18d8_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::holoviews-1.9.2-py27_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::holoviews-1.9.2-py35_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::holoviews-1.9.2-py36_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::holoviews-1.9.4-py27_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::holoviews-1.9.4-py35_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::holoviews-1.9.4-py36_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::holoviews-1.9.5-py27_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::holoviews-1.9.5-py35_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::holoviews-1.9.5-py36_0.tar.bz2
-    "param",
+    "param <2.0.0a0",
linux-64::hvplot-0.8.3-py310h06a4308_0.tar.bz2
-    "param >=1.9.0",
+    "param >=1.9.0,<2.0.0a0",
linux-64::hvplot-0.8.3-py311h06a4308_0.tar.bz2
-    "param >=1.9.0",
+    "param >=1.9.0,<2.0.0a0",
linux-64::hvplot-0.8.3-py38h06a4308_0.tar.bz2
-    "param >=1.9.0",
+    "param >=1.9.0,<2.0.0a0",
linux-64::hvplot-0.8.3-py39h06a4308_0.tar.bz2
-    "param >=1.9.0",
+    "param >=1.9.0,<2.0.0a0",
linux-64::hvplot-0.8.4-py310h06a4308_0.tar.bz2
-    "param >=1.9.0",
+    "param >=1.9.0,<2.0.0a0",
linux-64::hvplot-0.8.4-py311h06a4308_0.tar.bz2
-    "param >=1.9.0",
+    "param >=1.9.0,<2.0.0a0",
linux-64::hvplot-0.8.4-py38h06a4308_0.tar.bz2
-    "param >=1.9.0",
+    "param >=1.9.0,<2.0.0a0",
linux-64::hvplot-0.8.4-py39h06a4308_0.tar.bz2
-    "param >=1.9.0",
+    "param >=1.9.0,<2.0.0a0",
linux-64::panel-0.13.0-py310h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.13.0-py37h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.13.0-py38h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.13.0-py39h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.13.1-py310h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.13.1-py37h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.13.1-py38h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.13.1-py39h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.14.1-py310h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.14.1-py37h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.14.1-py38h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.14.1-py39h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.14.2-py310h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.14.2-py311h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.14.2-py37h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.14.2-py38h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.14.2-py39h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.14.3-py310h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.14.3-py311h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.14.3-py37h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.14.3-py38h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-0.14.3-py39h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.0.2-py310h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.0.2-py311h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.0.2-py38h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.0.2-py39h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.0.3-py310h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.0.3-py311h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.0.3-py38h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.0.3-py39h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.0.4-py310h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.0.4-py311h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.0.4-py38h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.0.4-py39h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.1.0-py310h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.1.0-py311h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.1.0-py38h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.1.0-py39h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.1.1-py310h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.1.1-py311h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.1.1-py38h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.1.1-py39h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.2.1-py310h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.2.1-py311h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.2.1-py38h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.2.1-py39h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.2.2-py310h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.2.2-py311h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.2.2-py38h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.2.2-py39h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.2.3-py310h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.2.3-py311h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.2.3-py38h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
linux-64::panel-1.2.3-py39h06a4308_0.tar.bz2
-    "param >=1.12.0",
+    "param >=1.12.0,<2.0.0a0",
Writing out new repodata as main/noarch/repodata-patched.json for 'noarch' platform.
New Hot Fixes:
noarch::datashader-0.10.0-py_0.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
noarch::datashader-0.11.0-py_0.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
noarch::datashader-0.11.1-py_0.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
noarch::datashader-0.12.1-pyhd3eb1b0_1.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
noarch::datashader-0.13.0-pyhd3eb1b0_1.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
noarch::datashader-0.6.8-py_0.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
noarch::datashader-0.6.9-py_0.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
noarch::datashader-0.7.0-py_0.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
noarch::datashader-0.8.0-py_0.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
noarch::datashader-0.9.0-py_0.tar.bz2
-    "param >=1.6.1",
+    "param >=1.6.1,<2.0.0a0",
noarch::panel-0.10.2-pyhd3eb1b0_0.tar.bz2
-    "param >=1.10.0",
+    "param >=1.10.0,<2.0.0a0",
noarch::panel-0.10.3-pyhd3eb1b0_0.tar.bz2
-    "param >=1.10.0",
+    "param >=1.10.0,<2.0.0a0",
noarch::panel-0.10.3-pyhd3eb1b0_1.tar.bz2
-    "param >=1.10.0",
+    "param >=1.10.0,<2.0.0a0",
noarch::panel-0.11.2-pyhd3eb1b0_0.tar.bz2
-    "param >=1.10.0",
+    "param >=1.10.0,<2.0.0a0",
noarch::panel-0.11.3-pyhd3eb1b0_0.tar.bz2
-    "param >=1.10.0",
+    "param >=1.10.0,<2.0.0a0",
noarch::panel-0.12.0-pyhd3eb1b0_0.tar.bz2
-    "param >=1.10.0",
+    "param >=1.10.0,<2.0.0a0",
noarch::panel-0.12.1-pyhd3eb1b0_0.tar.bz2
-    "param >=1.10.0",
+    "param >=1.10.0,<2.0.0a0",
noarch::panel-0.12.6-pyhd3eb1b0_0.tar.bz2
-    "param >=1.10.0",
+    "param >=1.10.0,<2.0.0a0",
noarch::panel-0.12.7-pyhd3eb1b0_0.tar.bz2
-    "param >=1.10.0",
+    "param >=1.10.0,<2.0.0a0",
noarch::panel-0.4.0-0.tar.bz2
-    "param >=1.8.2",
+    "param >=1.8.2,<2.0.0a0",
noarch::panel-0.5.1-0.tar.bz2
-    "param >=1.9.0",
+    "param >=1.9.0,<2.0.0a0",
noarch::panel-0.6.0-h39e3cac_0.tar.bz2
-    "param >=1.9.1",
+    "param >=1.9.1,<2.0.0a0",
noarch::panel-0.6.2-h39e3cac_0.tar.bz2
-    "param >=1.9.0",
+    "param >=1.9.0,<2.0.0a0",
noarch::panel-0.6.3-0.tar.bz2
-    "param >=1.9.1",
+    "param >=1.9.1,<2.0.0a0",
noarch::panel-0.6.4-0.tar.bz2
-    "param >=1.9.1",
+    "param >=1.9.1,<2.0.0a0",
noarch::panel-0.7.0-0.tar.bz2
-    "param >=1.9.2",
+    "param >=1.9.2,<2.0.0a0",
noarch::panel-0.8.0-0.tar.bz2
-    "param >=1.9.2",
+    "param >=1.9.2,<2.0.0a0",
noarch::panel-0.9.3-py_0.tar.bz2
-    "param >=1.9.2",
+    "param >=1.9.2,<2.0.0a0",
noarch::panel-0.9.5-py_1.tar.bz2
-    "param >=1.9.3",
+    "param >=1.9.3,<2.0.0a0",
noarch::panel-0.9.6-py_0.tar.bz2
-    "param >=1.9.3",
+    "param >=1.9.3,<2.0.0a0",
noarch::panel-0.9.7-py_0.tar.bz2
-    "param >=1.9.3",
+    "param >=1.9.3,<2.0.0a0",
```

</details>

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1218,6 +1218,45 @@ def patch_record_in_place(fn, record, subdir):
...
+        hvplot='0.8.4',
+        datashader='0.15.2',
+        geoviews='1.10.1',
+    )
+    for _holoviz_pkg, _holoviz_version in _holoviz_version_mapping.items():
+        if name != _holoviz_pkg:
+            continue
+        if VersionOrder(version) > VersionOrder(_holoviz_version):
+            continue
+        for i, dep in enumerate(depends):
```
Comment (ryanskeith) [inline main.py:1238]: I think this for loop could be replaced by the replace_dep function for the most part.  Though this works.

File: main.py [hunk 2]
```diff
@@ -1218,6 +1218,45 @@ def patch_record_in_place(fn, record, subdir):
...
+    # correctly.
+    _holoviz_version_mapping = dict(
+        panel='1.2.3',
+        holoviews='1.17.1',
+        hvplot='0.8.4',
+        datashader='0.15.2',
+        geoviews='1.10.1',
+    )
+    for _holoviz_pkg, _holoviz_version in _holoviz_version_mapping.items():
+        if name != _holoviz_pkg:
```
Comment (ryanskeith) [inline main.py:1234]: let's just skip the whole loop by saying the following before the for loop.
`if name in _holoviz_version_mapping:`

**Review by ryanskeith — COMMENTED** [review]
These look good to me.  Though I would prefer skipping the code block rather than going through the loop everytime.

File: main.py (hunk 1 above)
Comment (maximlt) [reply main.py]: Indeed but as I realized it couldn't replace everything I preferred to do it manually.

File: main.py (hunk 2 above)
Comment (maximlt) [reply main.py]: Good idea! Implemented in https://github.com/AnacondaRecipes/repodata-hotfixes/pull/210/commits/329456eb75ba9d7d586d3f22c7399ed177396cc6

**Comment by maximlt** [comment]
> These look good to me. Though I would prefer skipping the code block rather than going through the loop everytime.

Thanks for the review! I replied to your comments.

**Review by ryanskeith — APPROVED** [review]
Thanks for improving the code.

**Pull request merged**